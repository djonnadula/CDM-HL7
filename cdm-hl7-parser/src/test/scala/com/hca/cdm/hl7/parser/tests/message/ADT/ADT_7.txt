MSH|^~\&|EPIC_AKAR|AKAR|DBM||20160808071746|QLA7968|ADT^A01|EPIC_AKAR_ADT_233494|D|2.3||Epic_MountainQA|||||||AKA
EVN|A01|20160808071746||ADT_EVENT|QLA7968^GARRETT^BRYAN^^^^^^MTN A^^^^^AKAR|20160808071600
PID|1||123546|2258067|SOFTLY^KILLING^ME||20100504|F||W|32 GRIZZLEY LN^^ANCHORAGE^AK^99508^USA^P^^ANCHORAGE|ANCHORAGE|(801)252-3201^P^H^^^801^2523201||ENGLISH|S|LDS|700004079|777-77-7777||Softly^Jane^^|NOT HISPANIC||||||||N
ZPD|||||||N||||
PD1|||ALASKA REGIONAL HOSPITAL^^40001001|1477536316^CHURCH^EMILY^^^^^^NPI^^^^NPI~KCE9863^CHURCH^EMILY^^^^^^THREEFOUR^^^^THREEFOUR||||||||||
NK1|1|SOFTLY^JANE^^|19|32 Grizzley Ln^^ANCHORAGE^AK^99508^USA|(801)252-3201^^H^^^801^2523201||EC|||||||||||||||||||||||
NK1|2|||^^^^^USA|||EMP||||||NONE||||||||||||||||||||50696|N
PV1|1|I|AKARPED^L.PED359^L.PED359A^AKAR^^^^^^^|L|||1100199995^INPATIENT^ATTENDING^PHYSICIAN^^^^^NPI^^^^NPI~20001^INPATIENT^ATTENDING^PHYSICIAN^^^^^PROVID^^^^PROVID~ABC1231^INPATIENT^ATTENDING^PHYSICIAN^^^^^THREEFOUR^^^^THREEFOUR|1477536316^CHURCH^EMILY^^^^^^NPI^^^^NPI~KCE9863^CHURCH^EMILY^^^^^^THREEFOUR^^^^THREEFOUR||PED||||2||n|1100199995^INPATIENT^ATTENDING^PHYSICIAN^^^^^NPI^^^^NPI~20001^INPATIENT^ATTENDING^PHYSICIAN^^^^^PROVID^^^^PROVID~ABC1231^INPATIENT^ATTENDING^PHYSICIAN^^^^^THREEFOUR^^^^THREEFOUR||10138614|99|||||||||||||||||||||Adm^Conf|||20160808071600||||||10138614
PV2||PD||||||20160808071500|||||||||||||n|N|||||||||||||||||||||||
ZPV|||||||||||20160808071600||||||||||||||||
OBX|1|TX|ADMSRCDETAIL^POINT OF ORIGIN|1|2|||||||||20160808|||||||
OBX|2|TX|ACCSTAT1^ADT: PATIENT STATUS|1|Adm|||||||||20160808|||||||
OBX|3|TX|ACCSTAT2^HOSPITAL - ADMIT CONFIRMATION STATUS|1|Conf|||||||||20160808|||||||
CON|1|REV.CONSENT|||||||||RECEIVED||20160808071700||||||||
GT1|1|17087|SOFTLY^JANE^^||32 GRIZZLEY LN^^ANCHORAGE^AK^99508^USA^^^ANCHORAGE|(801)252-3201^^^^^801^2523201||19820509|F|Facility P/F|19|||||EBAY|^^^^^USA|||F|||||||||||||||||||||||||
ZG1|