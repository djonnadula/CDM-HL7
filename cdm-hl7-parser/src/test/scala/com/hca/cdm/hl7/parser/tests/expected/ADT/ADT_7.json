{
  "0000.COMN": {
    "sending_facility": "AKAR",
    "message_control_id": "EPIC_AKAR_ADT_233494",
    "medical_record_num": "123546",
    "medical_record_urn": "2258067",
    "patient_account_num": "700004079"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "EPIC_AKAR",
    "sending_facility": "AKAR",
    "receiving_application": "DBM",
    "date_time_of_message": "20160808071746",
    "security": "QLA7968",
    "message_type": {
      "message_code": "ADT",
      "trigger_event": "A01"
    },
    "message_control_id": "EPIC_AKAR_ADT_233494",
    "processing_id": "D",
    "version_id": "2.3",
    "continuation_pointer": "Epic_MountainQA",
    "message_profile_id": "AKA"
  },
  "0002.EVN": {
    "event_type_code": "A01",
    "recorded_date_time": "20160808071746",
    "event_reason_code": "ADT_EVENT",
    "operator_id": {
      "person_id": "QLA7968",
      "family_name": "GARRETT",
      "given_name": "BRYAN",
      "assigning_authority": "MTN A",
      "assigning_facility": "AKAR"
    },
    "event_occured": "20160808071600"
  },
  "0003.PID": {
    "set_id": "1",
    "patient_id_list": "123546",
    "alternate_patient_id": "2258067",
    "patient_name": {
      "family_name": "SOFTLY",
      "given_name": "KILLING",
      "second_and_further_given_names_or_initials_thereof": "ME"
    },
    "date_time_of_birth": "20100504",
    "administrative_sex": "F",
    "race": "W",
    "patient_address": {
      "street_address": "32 GRIZZLEY LN",
      "city": "ANCHORAGE",
      "state_or_province": "AK",
      "zip_or_postal_code": "99508",
      "country": "USA",
      "address_type": "P",
      "county_parish_code": "ANCHORAGE"
    },
    "county_code": "ANCHORAGE",
    "phone_num_home": {
      "telephone_num": "(801)252-3201",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "H",
      "area_city_code": "801",
      "local_num": "2523201"
    },
    "primary_language": "ENGLISH",
    "marital_status": "S",
    "religion": "LDS",
    "patient_account_num": "700004079",
    "ssn_num_patient": "777-77-7777",
    "mother_name": {
      "family_name": "Softly",
      "given_name": "Jane"
    },
    "ethnic_group": "NOT HISPANIC",
    "patient_death_indicator": "N"
  },
  "0004.ZPD": {
    "pregnancy_flag": "N"
  },
  "0005.PD1": {
    "patient_primary_facility": {
      "organization_name": "ALASKA REGIONAL HOSPITAL",
      "id_num": "40001001"
    },
    "patient_primary_care_provider_name_and_id_num": [
      {
        "person_id": "1477536316",
        "last_name": "CHURCH",
        "first_name": "EMILY",
        "assgn_authr": "NPI",
        "identifier_type_code": "NPI"
      },
      {
        "person_id": "KCE9863",
        "last_name": "CHURCH",
        "first_name": "EMILY",
        "assgn_authr": "THREEFOUR",
        "identifier_type_code": "THREEFOUR"
      }
    ]
  },
  "0006.NK1": {
    "set_id": "1",
    "name": {
      "family_name": "SOFTLY",
      "given_name": "JANE"
    },
    "relationship": "19",
    "address": {
      "street_address": "32 Grizzley Ln",
      "city": "ANCHORAGE",
      "state_or_province": "AK",
      "zip_or_postal_code": "99508",
      "country": "USA"
    },
    "phone_num": {
      "telephone_num": "(801)252-3201",
      "telecommunication_equipment_type": "H",
      "area_city_code": "801",
      "local_num": "2523201"
    },
    "contact_role": "EC"
  },
  "0007.NK1": {
    "set_id": "2",
    "address": {
      "country": "USA"
    },
    "contact_role": "EMP",
    "organization_name": "NONE",
    "next_of_kin_associated_party_ids": "50696",
    "job_status": "N"
  },
  "0008.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "AKARPED",
      "room": "L.PED359",
      "bed": "L.PED359A",
      "facility": "AKAR"
    },
    "admission_type": "L",
    "attending_doctor": [
      {
        "person_id": "1100199995",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "NPI",
        "id_type_code": "NPI"
      },
      {
        "person_id": "20001",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "PROVID",
        "id_type_code": "PROVID"
      },
      {
        "person_id": "ABC1231",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "referring_doctor": [
      {
        "person_id": "1477536316",
        "family_name": "CHURCH",
        "given_name": "EMILY",
        "assigning_authority": "NPI",
        "id_type_code": "NPI"
      },
      {
        "person_id": "KCE9863",
        "family_name": "CHURCH",
        "given_name": "EMILY",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "hospital_service": "PED",
    "admit_source": "2",
    "vip_indicator": "n",
    "admitting_doctor": [
      {
        "person_id": "1100199995",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "NPI",
        "id_type_code": "NPI"
      },
      {
        "person_id": "20001",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "PROVID",
        "id_type_code": "PROVID"
      },
      {
        "person_id": "ABC1231",
        "family_name": "INPATIENT",
        "given_name": "ATTENDING",
        "second_and_further_given_names_or_initials_thereof": "PHYSICIAN",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "visit_num": "10138614",
    "financial_class": "99",
    "account_status": {
      "id": "Adm",
      "text": "Conf"
    },
    "admit_date_time": "20160808071600",
    "alternate_visit_id": "10138614"
  },
  "0009.PV2": {
    "accommodation_code": "PD",
    "expected_admit_date_time": "20160808071500",
    "visit_publicity_code": "n",
    "visit_protection_indicator": "N"
  },
  "0010.ZPV": {
    "first_inpatient_occurence_date_time": "20160808071600"
  },
  "0011.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_id": {
      "id": "ADMSRCDETAIL",
      "text": "POINT OF ORIGIN"
    },
    "obsv_sub_id": "1",
    "obsv_value": "2",
    "date_time_of_the_obsv": "20160808"
  },
  "0012.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_id": {
      "id": "ACCSTAT1",
      "text": "ADT: PATIENT STATUS"
    },
    "obsv_sub_id": "1",
    "obsv_value": "Adm",
    "date_time_of_the_obsv": "20160808"
  },
  "0013.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_id": {
      "id": "ACCSTAT2",
      "text": "HOSPITAL - ADMIT CONFIRMATION STATUS"
    },
    "obsv_sub_id": "1",
    "obsv_value": "Conf",
    "date_time_of_the_obsv": "20160808"
  },
  "0014.CON": {
    "set_id": "1",
    "consent_type": "REV.CONSENT",
    "consent_status": "RECEIVED",
    "consent_decision_date_time": "20160808071700"
  },
  "0015.GT1": {
    "set_id": "1",
    "guarantor_num": "17087",
    "guarantor_name": {
      "family_name": "SOFTLY",
      "given_name": "JANE"
    },
    "guarantor_address": {
      "street_address": "32 GRIZZLEY LN",
      "city": "ANCHORAGE",
      "state_or_province": "AK",
      "zip_or_postal_code": "99508",
      "country": "USA",
      "county_parish_code": "ANCHORAGE"
    },
    "guarantor_phone_num_home": {
      "telephone_num": "(801)252-3201",
      "area_city_code": "801",
      "local_num": "2523201"
    },
    "guarantor_date_time_of_birth": "19820509",
    "guarantor_administrative_sex": "F",
    "guarantor_type": "Facility P/F",
    "guarantor_relationship": "19",
    "guarantor_employer_name": "EBAY",
    "guarantor_employer_address": {
      "country": "USA"
    },
    "guarantor_employment_status": "F"
  },
  "0016.ZG1": {
    "fpl_effective_from_date": "ZG1"
  }
}